components/cluster/pcs/Makefile

```diff
@@ -14,6 +14,12 @@
 
 include ../../../make-rules/shared-macros.mk
 
+RUBY_VERSION = 2.2
+RUBY_LIB_VERSION = 2.2.0
+RUBY.2.2 = /usr/ruby/2.2/bin/ruby
+RUBY = $(RUBY.$(RUBY_VERSION))
+RUBY_VERSIONS = $(RUBY_LIB_VERSION)
```

Comment:
@xenol I think we should add part of these definitions to shared-macros.mk, and here use just RUBY_VERSION=2.2